middleware/transaction/source/manager/admin/server.cpp

```diff
@@ -64,8 +64,8 @@ namespace casual
                         common::service::visibility::Type::undiscoverable,
                         common::service::category::admin
                      },
-                     { service::name::scale::instances,
-                        std::bind( &local::scale::instances, std::placeholders::_1, std::ref( state)),
+                     { service::name::scale::resource::proxies,
+                        std::bind( &local::scale::resource::proxy::instances, std::placeholders::_1, std::ref( state)),
```

Comment:
You could get rid of `std::bind`, which sorta has played out it's role and replaced with generic lambdas. The placeholder stuff is not something to write home abut...